azure/Kqlmagic/adal_token_cache.py

```diff
@@ -117,14 +148,45 @@ def _query_cache(self, is_mrrt, user_id, client_id):
                 matches.append(v)
         return matches
 
+    def clear_db(self):
+
+        items = self.db.items()
+
+        for token_tuple in items:
+
+            db_key = token_tuple[0] # 'kqlmagicstore/tokens/USER_ID' address
+            data = token_tuple[1] #encrypted token
+            time_created, _unused = Fernet._get_unverified_token_data(data)  #get token timestamp 
```

Comment:
I think that theoretically the db can contain a null value, please verify it works if data is None ore empty.
in addition I think you should checkthat the data is a fernet token, just to be on the safe side (you can check that the metadata makes sense). If it is not a fernet token you can delete it, but send a message to the user.